charts/orchestrator-k8s/hack/bump_chart_version.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Get the directory of the script
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+CHART="${SCRIPT_DIR}/../Charts.yaml" "${SCRIPT_DIR}/../../../hack/bump_chart_version.sh"
+
```

Comment:
there is a need to pass also the argument to the script.

Reply:
this is mostly interactive I guess?

Reply:
no, it expects any of the arguments, and bump it to the next close version according to the argument,. e.g.:

0.2.1 --> 0.2.2 for patch:
0.2.1 --> 0.3.0 for minor:
0.2.1 --> 1.0.0 for major:

```
→ ./hack/bump_chart_version.sh 
Error: Incorrect number of arguments
Usage: ./hack/bump_chart_version.sh <flag>
Flags:
  --bump-minor-version       : Bump minor version and set patch to 0
  --bump-major-version       : Bump major version
  --bump-patch-version       : Bump patch version
  --bump-minor-app-version   : Bump minor app version and set patch to 0
  --bump-major-app-version   : Bump major app version
  --bump-patch-app-version   : Bump patch app version

```

Reply:
ack, changed

Reply:
hold - I need to fix the bump script